packages/fxa-settings/src/models/Account.ts

```diff
@@ -550,11 +550,26 @@ export class Account implements AccountData {
         `,
         variables: { input: { email } },
       });
-
       return result.data.passwordForgotSendCode;
     } catch (err) {
-      const errno = (err as ApolloError).graphQLErrors[0].extensions?.errno;
-      if (errno && AuthUiErrorNos[errno]) {
+      const graphQlError = (err as ApolloError).graphQLErrors[0];
+      const errno = graphQlError.extensions?.errno;
+      if (
+        errno &&
+        AuthUiErrorNos[errno] &&
+        errno === AuthUiErrors.THROTTLED.errno
+      ) {
+        const throttledErrorWithRetryAfter = {
+          ...AuthUiErrorNos[errno],
+          retryAfter: graphQlError.extensions?.retryAfter,
+          retryAfterLocalized: graphQlError.extensions?.retryAfterLocalized,
+        };
+        throw throttledErrorWithRetryAfter;
```

Comment:
Q: Should this throw a new `ThrottledError` instance versus throwing an Object?

Reply:
The error object here is consistent with `AuthUiError` objects thrown by other Account methods.